DuckDuckGoTests/URLExtensionTests.swift

```diff
@@ -21,7 +21,15 @@
 import XCTest
 
 class URLExtensionTests: XCTestCase {
-    
+
+    func testUrlWithUserIsFalse() {
```

Comment:
How do you feel about the `When<condition>Then<expectedResult>` syntax for test names? I use it a fair bit - but not all the time as it doesn't always flow

Reply:
Yep, I'm good with this. Will endeavour to do this automatically moving forward.